.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+Describe the changes in this PR.
+
+## Related issues
+Addresses [issue #].
+
+## Testing
+Describe how this change was tested.
+
+## FHIR Team Checklist
+- [ ] **Update the title** of the PR to be succinct and less than 50 characters
+- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
+Update the title of the PR to be succinct and less than 50 characters
+- [ ] Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
+- [ ] Tag the PR with **Azure API for FHIR** if this will release to the managed service
```

Comment:
>if this will release to the managed service [](start = 45, length = 43)

so basically everything which doesn't involve docs or build?

Reply:
Yes, also feature flagged work and sql